tests/testthat/test-transform_point_model_out.R

```diff
@@ -43,24 +43,22 @@ test_that("model_out_tbl_1 output is valid", {
 })
 
 
-test_that("model_out_tbl_1 columns are valid", {
+cli::test_that_cli("transform_point_model_out() throws warning for unexpected columns", {
   # Task IDs: location, reference_date, target_end_date, target
 
   model_out_tbl_1 <- utils::read.csv(test_path("testdata/model_out_tbl_point_1.csv")) |>
     dplyr::rename(loc = location, trgt = target, date = target_end_date)
 
   target_observations_1 <- utils::read.csv(test_path("testdata/target_data_1.csv")) |>
     dplyr::rename(date = target_end_date) # leave location column unchanged
-  suppressWarnings({
-    expect_warning(
-      transform_point_model_out(
-        model_out_tbl = model_out_tbl_1,
-        target_observations = target_observations_1,
-        output_type = "mean"
-      ),
-      regexp = "unexpected column"
+
+  expect_snapshot(
+    transform_point_model_out(
+      model_out_tbl = model_out_tbl_1,
+      target_observations = target_observations_1,
+      output_type = "mean"
     )
-  })
+  )
```

Comment:
Oh I see now why this test was failing! `cli::cli_alert_warning()` produces a stylized `message` and `cli::cli_warn()` produces an actual warning... I had gotten them confused.

Reply:
i hadn't actually seen `cli_warn` -- good to know about that one.  regardless, this is now an error...